### PR TITLE
Test env: add Django 1.9 to travis, adjust cms32 dependency

### PR DESCRIPTION
**Pull request by cyberstar**
``flake8`` ``max_length`` should be enabled after cleaning up the code.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
+    cms32: https://github.com/divio/django-cms/archive/release/3.2.x.zip
     -r{toxinidir}/requirements-test.txt
 
-[testenv:pep8]
+[testenv:flake8]
 deps = flake8
 commands = flake8
 skip_install = true
 
+[flake8]
```
Comment (yakky) [inline tox.ini]: how about setting this in setup.cfg?

File: .travis.yml [hunk 2]
```diff
@@ -5,8 +5,13 @@ sudo: false
 
 env:
   matrix:
+    - TOXENV=pep8
+    - TOXENV=py35-django19-cms32
     - TOXENV=py35-django18-cms32
     - TOXENV=py35-django18-cms31
+    - TOXENV=py34-django19-cms32
+    - TOXENV=py33-django19-cms32
```
Comment (yakky) [inline .travis.yml]: Python 3.3 is not supported by django 1.9

File: tox.ini (hunk 1 above)
Comment (cyberstar) [inline tox.ini]: If that change is not critical I'd prefer to use the same approach as on essential add-ons.
Also I've encountered a bug where flake uses any ``setup.cfg`` which is accessible through ``PATH`` (i.e. from home directory)

Comment (yakky) [inline tox.ini]: Fine

**Pull request merged**